docs/_integration-manager/camel-routes/camel-route-azure.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Camel Route Configuration for Azure Blob Storage
+nav_order: 3
+parent: Camel Route Configuration
+---
+# Camel Route Configuration for Azure Blob Storage
+
+## Overview
+
```

Comment:
## Overview

Monitor an Azure Blob Container folder for new files and submit to the JobConfig run API

Reply:
Done.